_internal/DeerLister.php

```diff
@@ -153,6 +153,7 @@ private function readDirectory(string $directory, mixed $config): array|false
             array_push($files,
                 [
                     "name" => $name,
+                    "share" => urlencode($name),
```

Comment:
This seems redundant, we already have the name and url encoding it should be done by the link builder.